test/integrationtests/skills/skill_tester.py

```diff
@@ -519,9 +529,7 @@ def __init__(self, test_case, skill=None):
         _x = ['and']
         if 'utterance' in test_case and 'intent_type' in test_case:
```

Comment:
This will highlight my own ignorance, but would it be a valid test at all if 'utterance' doesn't exist?

Reply:
intents can only be tested if there is an utterance. The tests for questions and common play handlers don't require an utterance in the same way.